spec/spec_helper.rb

```diff
@@ -15,3 +25,10 @@
     puts "\e[33mTry it now! Just run: rubocop\e[0m"
   end
 end
+
+#
```

Comment:
How comes this bit is commented out? How does your test database work without it?